### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Extensions - Uninstall_VoiceOver: When the user uses Voice-over arrow keys to move across the 'Disable', 'Uninstall', 'Manage' button, the voiceover announces about a control that is not visible to the users on the screen but has menu items inside it when accessed.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19895]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Voiceover;#AINotInScope;#WCAG1.4.4;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4
Screen Reader: Voice-Over

## Repro Steps: 

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'Extensions' button and select an extension. (Example: SQL database projects)
5. Tab till 'Disable' button.
6. Turn on Voice-over and use arrow keys to move across the controls.

## Actual:
When the user uses Voice-over arrow keys to move across the 'Disable', 'Uninstall', 'Manage' button, the voiceover announces about a control that is not visible to the users on the screen but has menu items inside it when accessed.

## Expected:
The control should be visible to the users properly.

## User Impact:
 keyboard users will not get to view the data and screen reader users will not know what the items are present.

## Attachment:
![Presentation1111](https://user-images.githubusercontent.com/98393966/176598043-58282646-4982-4af6-a1fc-b64f7b2365f2.gif)

**Comment by alanrenmsft** [comment]
@bbarik06 fix should be included in tomorrow's insider build. please retest

**Comment by alanrenmsft** [comment]
please let us know if the fix is not working. closing since the SLA is near.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.2
![image](https://user-images.githubusercontent.com/85863999/220018106-4ddfcad9-b4e4-4ae8-90c8-8d8796072267.png)
